platform/include/encoding.h

```diff
@@ -0,0 +1,2832 @@
+/* See LICENSE for license details. */
+
+#ifndef RISCV_CSR_ENCODING_H
+#define RISCV_CSR_ENCODING_H
+
```

Comment:
Comments missing

Reply:
Thanks for the catch. Will focus more on documentation moving forward...